src/abstract_hardware_model.h

```diff
@@ -1081,6 +1086,10 @@ class warp_inst_t : public inst_t {
   void issue(const active_mask_t &mask, unsigned warp_id,
              unsigned long long cycle, int dynamic_warp_id, int sch_id);
 
+  void issue(const active_mask_t &mask, unsigned warp_id,
+             unsigned long long cycle, int dynamic_warp_id, int sch_id,
+             unsigned long long streamID);
```

Comment:
Require a StreamID always. No need to keep the basic issue() without the streamID.

Reply:
Addressed in commit e8cc02d